tests/apichecks/testdata/exceptions/acronyms.txt

```diff
@@ -0,0 +1,508 @@
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].accessConfig[].natIpRef.name" should be ".spec.networkInterface[].accessConfig[].natIPRef.name"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].accessConfig[].natIpRef.namespace" should be ".spec.networkInterface[].accessConfig[].natIPRef.namespace"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].aliasIpRange" should be ".spec.networkInterface[].aliasIPRange"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].aliasIpRange[]" should be ".spec.networkInterface[].aliasIPRange[]"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].aliasIpRange[].ipCidrRange" should be ".spec.networkInterface[].aliasIPRange[].ipCIDRRange"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].aliasIpRange[].subnetworkRangeName" should be ".spec.networkInterface[].aliasIPRange[].subnetworkRangeName"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].internalIpv6PrefixLength" should be ".spec.networkInterface[].internalIPV6PrefixLength"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPV6"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].ipv6AccessConfig[].externalIpv6PrefixLength" should be ".spec.networkInterface[].ipv6AccessConfig[].externalIPV6PrefixLength"
+[acronyms] crd=computeinstancetemplates.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.networkInterface[].networkIp" should be ".spec.networkInterface[].networkIP"
+[acronyms] crd=computeinterconnectattachments.compute.cnrm.cloud.google.com version=v1beta1: field ".status.cloudRouterIpAddress" should be ".status.cloudRouterIPAddress"
+[acronyms] crd=computeinterconnectattachments.compute.cnrm.cloud.google.com version=v1beta1: field ".status.customerRouterIpAddress" should be ".status.customerRouterIPAddress"
+[acronyms] crd=computeinterconnectattachments.compute.cnrm.cloud.google.com version=v1beta1: field ".status.googleReferenceId" should be ".status.googleReferenceID"
+[acronyms] crd=computemanagedsslcertificates.compute.cnrm.cloud.google.com version=v1alpha1: field ".status.certificateId" should be ".status.certificateID"
```

Comment:
So are we meant to chip away at these exceptions as a team? For instance, would we want to correct this field if we had the chance?

Reply:
For beta: I don't think we can do anything about them.

For alpha: I would say yes, but probably not a priority unless we're already touching the resource.

The big thing is that we want to avoid introducing new exceptions if we can.  So the idea is that if you're adding an exception, it's not possible to PR that without it showing up as a diff here (otherwise it fails tests).  So it should draw the PR reviewer's attention to it, and thus it makes it easy to have the discussion.

Specifically for `Id` vs `ID` I don't know.  KRM conventions say `ID`, but we have done `Id` pretty consistently.  We might argue that `Id` is more "GCP API", and thus more appropriate for KCC.

I'd say we don't need to do anything yet for the existing field exception list.  But as we touch the alpha resources, or as we introduce new resources, that's when we should think twice about any new exceptions (or any acronyms that are missing from the "API linter rule"!)

cc @yuwenma

Reply:
I really like this way to guide the developers! Let's merge this in and turn on the check!